types/react-faux-dom/index.d.ts

```diff
@@ -19,20 +20,18 @@ export const defaultView: {
     getComputedStyle(node: Element): { getPropertyValue(name: string): string };
 };
 
-export namespace mixins {
-    const core: any;
-    const anim: any;
-}
-
 export function createElement(nodeName: string): Element;
 
 export function createElementNS(namespace: string, nodeName: string): Element;
 
 export function compareDocumentPosition(): number;
 
 export interface ReactFauxDomProps {
-    connectFauxDOM?(node: string, name: string, discardNode?: any): Element;
-    animateFauxDOM?(duration: number): void;
+    connectFauxDOM(node: string, name: string, discardNode?: any): Element;
```

Comment:
Why remove the ? handles some edge cases and was added explicitely

Reply:
Edge cases like? Could you look into the attached [source code](https://github.com/Olical/react-faux-dom/blob/master/lib/_withFauxDOM.js)?

I honestly haven't found a way to not pass props using higher order component as it's just always passed in there as a prop. Mixins were removed with one of the previous releases.

Reply:
@cleverguy25 - could you answer my question? 😄